README.md

```diff
@@ -1,19 +1,90 @@
 # Workflows
 
-The repo for Workflows that appear within Warp and within [commands.dev](https://www.commands.dev/)
+The repo for all _global_ Workflows that appear within Warp and within [commands.dev](https://www.commands.dev/).
+
+**To learn how to create local or repository workflows, see [our docs](https://github.com/warpdotdev/warp-internal/pull/2205).**
+
+<img width="736" alt="image" src="https://user-images.githubusercontent.com/4110292/164031239-49f0ec9e-f124-44c4-89e6-6facc9bf9a8f.png">
+
 
 ## What are Workflows?
 
 Workflows are an easier way to execute and share commands within Warp. They are searchable by name, description, or command and are easily parameterized. See our documentation for more details: [https://docs.warp.dev/features/workflows](https://docs.warp.dev/features/workflows)
 
-## How Do I Access Workflows?
+## How Do I Access Workflows within Warp?
 
-Workflows can be accessed within Warp directly within the app, either through the Command Palette or by pressing `ctrl-shift-r`.
+Workflows can be accessed directly within Warp, either through the Command Palette or by pressing `ctrl-shift-r`.
 
 All public workflows (i.e. workflows within this repo) are also available at [commands.dev](https://www.commands.dev/).
 
 ## Contributing
-
 Contributions are always welcome! If you have a workflow that would be useful to many Warp users, feel free to send a PR to add a Workflow spec.
 
-The existing preset files should serve as an example for the format, while the a more comprehensive description of the file format is available in [FORMAT.md](FORMAT.md). Please request a review from someone on the Warp team once a PR is ready for review.
+All workflows are defined as YAML files within the [`specs/`](specs/) directory. 
+
+### File Format
+A comprehensive description of the file format is available in [FORMAT.md](FORMAT.md).
+Additionally, see the workflow below as an example to quickly get started:
+
+```yaml
+---
+# The name of the workflow.
+name: Uninstall a Homebrew package and all of its dependencies
+# The corresponding command for the workflow. Any arguments should be surrounded with two curly braces. E.g `command {{arg}}`.
+command: "brew tap beeftornado/rmtree\nbrew rmtree {{package_name}}"
+# Any tags that workflow should be categorized with.
```

Comment:
Do we have any restrictions on what tags can/should be added here (for global/public workflows)?  I expect we don't want the in-app list to grow too long (it would, in the extreme, make search the only way to reasonably find workflows in-app).

Reply:
yeah I'm adding that as a section in the "what makes a good workflow" section--ideally we don't want too many unnecessary tags